src/managers/WebhookManager.js

```diff
@@ -340,15 +343,15 @@ class WebhookManager {
 
   _compileJsonTemplate = (template?: ?Object, variables: Object): ?Object => {
     if (!template) {
-      return;
+      return undefined;
```

Comment:
with `null`, it breaks tests.  I can do WEBHOOK_BASE = { every_not_defined_prop: null }, and use null everywhere in the tests though.. but seems undefined more like the truth.
2) or I can do `return;`  and just disable `eslint-rule consistent-return` in this method.